setup.py

```diff
@@ -9,7 +9,7 @@
 
 setup(
     name='urbansim_parcels',
-    version='0.1dev',
+    version='0.1dev1',
```

Comment:
Should have another dot to match `__init__.py` -- 0.1.dev1

Reply:
Solved in https://github.com/urbansim/urbansim_parcels/pull/32/commits/2a23d734298cb1650a80beb45d41acb88d8371d1